setup_win32.py

```diff
@@ -175,7 +175,8 @@ def run(self):
         shutil.rmtree(build_exe.build_exe)
 
 
-setup(name='pdfarranger',
+setup(name='PDF Arranger',
+      author='PDF-Arranger developer team',
```

Comment:
Maybe you can refer to this file: https://github.com/pdfarranger/pdfarranger/blob/d7d0c52be6076fe3e5964f8b6e4877103d665eec/setup.py#L108

Reply:
> [!WARNING]
>
> Rejected by https://github.com/pdfarranger/pdfarranger/pull/1134#issuecomment-2429296836

Reply:
```suggestion
      author='The PDF Arranger team',
```

Suggested by https://github.com/pdfarranger/pdfarranger/pull/1134#issuecomment-2429296836.